src/overlays/actors/ovl_En_Guest/z_en_guest.c

```diff
@@ -8,7 +9,20 @@ void EnGuest_Init(Actor* thisx, GlobalContext* globalCtx);
 void EnGuest_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnGuest_Update(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void func_80A5046C(EnGuest* this);
+void func_80A50518(EnGuest* this, GlobalContext* globalCtx);
+void func_80A5057C(EnGuest* this, GlobalContext* globalCtx);
+void func_80A505CC(Actor* thisx, GlobalContext* globalCtx);
+s32 EnGuest_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* actor);
+void EnGuest_Draw(Actor* thisx, GlobalContext* globalCtx);
+
+extern SkeletonHeader D_060000F0;
+extern AnimationHeader D_060042AC;
+extern UNK_TYPE D_060005FC;
+extern UNK_TYPE D_060006FC;
+extern UNK_TYPE D_060007FC;
+extern Gfx D_060059B0;
```

Comment:
Display lists are always Gfx arrays (even if they only have one entry, which is probably not the case here, but just to be clear).
```suggestion
extern Gfx D_060059B0[];
```